tests/unit/test_file_handle_cache.py

```diff
@@ -0,0 +1,52 @@
+"""
+Tests the file handle cache
+"""
+
+from ga4gh import datamodel
+
+import unittest
+import tempfile
+import os
+
+import ga4gh.frontend as frontend
+
+
+class TestFileHandleCache(datamodel.PysamFileHandleCache, unittest.TestCase):
```

Comment:
Why do we need to inherit from the `PysamFileHandleCache` here? I would have thought it would be simpler to just create some instances.